telemetry/telemetry.go

```diff
@@ -112,11 +114,10 @@ func main() {
 			}
 			certificate, err = tls.LoadX509KeyPair(*serverCert, *serverKey)
 			if err != nil {
-				currentTime := time.Now().UTC()
-				log.Infof("Server Cert md5 checksum: %x at time %s", md5.Sum([]byte(*serverCert)), currentTime.String())
-				log.Infof("Server Key md5 checksum: %x at time %s", md5.Sum([]byte(*serverKey)), currentTime.String())
 				log.Exitf("could not load server key pair: %s", err)
 			}
+			validateSHA256(*serverCert)
+			validateSHA256(*serverKey)
```

Comment:
Original implementation is: when tls.LoadX509KeyPair has error, we generate log with md5 checksum of file name.
New implementation is: when tls.LoadX509KeyPair has no error, we generate log with sha256 checksum of file content.
Can you explain why?

Reply:
Should be generating log with sha512 checksum when loadx509 fails. Fixed in https://github.com/sonic-net/sonic-gnmi/pull/186